requirements-dev.txt

```diff
@@ -0,0 +1,71 @@
+-i https://pypi.org/simple
+alabaster==0.7.12
+attrs==22.1.0 ; python_version >= '3.5'
```

Comment:
small comment: I noticed that there's a space added in the requirements files before the ";" character.
Running ```pipenv requirements``` and ```pipenv requirements --dev-only```, with ```pipenv, version 2022.8.24``` I did not get this extra space character.
This currently does not affect anything, but if we add in the future a check in tox to see the diff of the current requirements file and the ```pipenv requirements``` command (to know if the requirements are updated), we will get that they are not the same file.

Reply:
Other than the above comment, everything else looks good to me.

Reply:
I filed a new issue for it. Good first issue btw
https://github.com/OwnCA/ownca/issues/81

Reply:
If it's ok, please approve the review ;)

Reply:
Changes are approved, but I think the "Review required" will not be checked ✅ since it notes: "At least 1 approving review is required by reviewers with write access."

Reply:
Welcome to the maintainer's group.
@lawndoc and @KAUTH are maintainers of OwnCA.

Reply:
https://pypi.org/project/OwnCA/0.3.3/